ci/jjb/jobs/projects.yaml

```diff
@@ -41,8 +41,9 @@
     - 2.16:
         pulp_version: '2.16'
     os:
-    - 'f27'
-    - 'rhel7'
+    - f27
+    - rhel7
+    script-var: ''
```

Comment:
What does `script-var: ''` do?

Reply:
script-var is a place-holder for changing the files according to the OS .

If the OS is anything else other than **f28**, the pulp-dev projects load the script pulp-install.sh.
If it is equal to f28, the pulp-dev projects load pulp-install-f28.sh ( which has SELinux permissive and also ansible py interpreter=3).
Thus the scripts are loaded dynamically